demo/views/chrome/footer/footer.js

```diff
@@ -1,26 +1,46 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import I18n from 'i18n-js';
+import './footer.scss';
 
 // Carbon
 import Icon from 'components/icon';
-import Link from 'components/link';
 
 // Demo
 import Wrapper from './../../common/wrapper';
 
 class Footer extends React.Component {
   render () {
     return (
-      <div className="demo-footer__background">
+      <div className='demo-footer__background'>
         <Wrapper>
           <div className='demo-footer'>
-            <div className="demo-footer__section demo-footer__social">
-              <Link href='https://twitter.com/sagegroupplc' target="_blank"><Icon type='twitter' /></Link>
-              <Link href='https://dribbble.com/sageuxdesign' target="_blank"><Icon type='dribbble' /></Link>
-              <Link href='https://github.com/Sage/carbon' target="_blank"><Icon type='github' /></Link>
+            <div className='demo-footer__section demo-footer__social'>
+              <a
+                href='https://twitter.com/sagegroupplc'
+                target='_blank'
+                className='demo-footer__link'
+                rel='noopener noreferrer'
+              >
+                <Icon type='twitter' />
+              </a>
+              <a
+                href='https://dribbble.com/sageuxdesign'
+                target='_blank'
+                rel='noopener noreferrer'
+                className='demo-footer__link'
+              >
+                <Icon type='dribbble' />
+              </a>
+              <a
+                href='https://github.com/Sage/carbon'
+                target='_blank'
+                rel='noopener noreferrer'
+                className='demo-footer__link'
+              >
+                <Icon type='github' />
+              </a>
```

Comment:
why was this changed? for the `rel` attr? If so you do you not just need to whitelist this prop as safe? https://github.com/Sage/carbon/blob/master/src/components/link/link.js#L153

(but also is not unrelated to this PR?)

Reply:
It was to fix a CSS specificity issue when running the demo site locally, meaning the footer links were the wrong colour.

ESLint complained about the lack of a `rel` attribute on external links, so I added that as well for good measure.